src/main/java/appeng/core/api/definitions/ApiParts.java

```diff
@@ -57,6 +57,7 @@
 	private final IItemDefinition iface;
 	private final IItemDefinition fluidIface;
 	private final IItemDefinition levelEmitter;
+	private final IItemDefinition fluidLevelEmitter;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "fluidLevelEmitter" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)